pom.xml

```diff
@@ -160,6 +160,12 @@
       <version>1.9.13</version>
     </dependency>
     <dependency>
+      <!-- bundle: Jackson-core -->
```

Comment:
This comment should be the same as the bundle ID as specified in the `category.xml` and `feature.xml`